src/polkadot/events/index.ts

```diff
@@ -0,0 +1,39 @@
+export enum PolkadotEvent {
+  OnBalanceChange = "onBalanceChange"
+}
+
+export interface EventEmitter<K = keyof PolkadotEvent>  {
+  addListener(event: K, listener: EventCallback): this;
```

Comment:
@mpetrunic Do you think I should change naming narrative from _listener_ to _callback_, for example: `addListener` => `addCallback` ?

Reply:
this is fine. You could install some library for event emitting